orchagent/neighorch.cpp

```diff
@@ -1420,6 +1420,17 @@ void NeighOrch::doVoqSystemNeighTask(Consumer &consumer)
                     {
                         SWSS_LOG_NOTICE("VOQ encap index updated for neighbor %s", kfvKey(t).c_str());
                         it = consumer.m_toSync.erase(it);
+
+                       /* Remove remaining DEL operation in m_toSync for the same neighbor.
+                        * Since DEL operation is supposed to be executed before SET for the same neighbor
+                        * A remaining DEL after the SET operation means the DEL operation failed previously and should not be executed anymore
+                        */
+                       auto rit = make_reverse_iterator(it);
```

Comment:
sign, we should have tab space aligned. @arlakshm , @ysmanman . I feel being insulted. @prsunny

Reply:
> sigh, we should have tab space aligned. @arlakshm , @ysmanman . I feel being insulted. @prsunny

I can raise a PR to fix the indentation.